oneflux_steps/ustar_cp_refactor_wip/cpdEvaluate_functions/initializeStatistics.m

```diff
@@ -0,0 +1,24 @@
+function [Stats2, Stats3] = initializeStatistics(nSeasons, nStrataX, varargin)
+
+	StatsMT=[]; 
+	StatsMT.n=NaN; StatsMT.Cp=NaN; StatsMT.Fmax=NaN; StatsMT.p=NaN; 
+	StatsMT.b0=NaN; StatsMT.b1=NaN; StatsMT.b2=NaN; StatsMT.c2=NaN; 
+	StatsMT.cib0=NaN; StatsMT.cib1=NaN; StatsMT.cic2=NaN; % preceding elements output by changepoint function
+	StatsMT.mt=NaN; StatsMT.ti=NaN; StatsMT.tf=NaN; 
```

Comment:
This is duplicate code between here and cpdBootstrap where this stuff was already modularised and tested. Best to use those functions.

Reply:
Thanks, updated it/

Reply:
Thanks, updated it.